group_vars/all.yaml

```diff
@@ -75,3 +75,7 @@ koji_db_data_dir: "{{ osbs_box_data_dir }}/pv/koji-db-data"
 registry_data_dir: "{{ osbs_box_data_dir }}/pv/registry-data"
 cachito_archives_dir: "{{ osbs_box_data_dir }}/pv/cachito-archives"
 cachito_athens_dir: "{{ osbs_box_data_dir }}/pv/cachito-athens-storage"
+
+docker_id: mylogin
```

Comment:
Maybe `dockerhub_` ?

Reply:
That's literally Docker's name for it... WDYT?

Reply:
ok then